src/app/common/modals/modals.less

```diff
@@ -453,6 +453,186 @@
     }
 }
 
+.tabswitcher-modal {
+    width: 452px;
+    min-height: 384px;
+
+    .wave-modal-content {
+        .wave-modal-body {
+            display: flex;
+            padding: 0px;
+            flex-direction: column;
+            align-items: flex-start;
+            align-self: stretch;
+            width: 100%;
+
+            .textfield-wrapper {
+                padding: 20px 20px 0px;
+
+                .wave-input-decoration.start-position {
+                    height: 100%;
+
+                    .tabswitcher-search-prefix {
+                        opacity: 0.5;
+                        font-size: 13px;
+                    }
+                }
+            }
+
+            .list-container {
+                overflow: hidden;
+                padding: 10px 0 20px;
+                width: 100%;
+            }
+
+            .list-container-inner {
+                width: 100%;
+                max-height: 300px;
+                overflow-y: hidden;
+                padding: 0 20px;
+
+                &:focus,
+                &:hover {
+                    overflow-y: overlay;
+                    padding-right: 16px;
+                }
+
+                .options-list {
+                    width: 100%;
+
+                    .search-option {
+                        padding: 5px 5px 5px 15px;
+                        display: flex;
+                        align-items: center;
+                        border: 1px solid transparent;
+
+                        &:hover {
+                            border: 1px solid rgba(241, 246, 243, 0.15);
+                            border-radius: 4px;
+                            background: rgba(255, 255, 255, 0.06);
+                        }
+
+                        span {
+                            display: flex;
+                        }
+
+                        span:first-child {
+                            width: 20px;
+                        }
+                    }
+
+                    .selected-option {
+                        border: 1px solid rgba(241, 246, 243, 0.15);
+                        border-radius: 4px;
+                        background: rgba(255, 255, 255, 0.06);
+                    }
+
+                    .color-green,
```

Comment:
i think all of these `color` classes should be combined into app.less.  we already have rules for .icon.color-X in there that will set the path fill color.  just add the `i { color: ...}` rules there as well.  (i think we also need color-default there as well).  then maybe add the `icon` class to your renderIcon output and we'll be set and can remove all of these duplicate classes.

Reply:
resolved